### PR TITLE
Run PEP8 on content

### PR DESCRIPTION
**Pull request by sptramer**
Hello! The Azure documentation team (Content & Learning) is updating existing
    Python samples to conform to PEP8. This consists of (only) running `autopep8`
    on content. Even if this PR contains only whitespace changes, please accept it.
    # 
    If you have questions about the rationale behind this change, please email
    Microsoft alias `sttramer`.

### CODE REVIEW

**Comment by dtzar** [comment]
Thanks @sptramer - We'd like to setup CI to enforce this style as well. Do you have an example of using `pylint` or `flake8` to enforce the PEP8 style by chance?

**Comment by sptramer** [comment]
I was just using `autopep8`, which seems to be pretty good at it. It has more aggressive PEP8 enforcement as well, but the defaults seem nice.

File: aml_service/pipelines/train_pipeline.py [hunk 1]
```diff
@@ -1,164 +1,164 @@
...
+import sys
+import os
+import time
+import click
+import json
+
+from azureml.pipeline.core import Pipeline
+from azureml.pipeline.steps import DatabricksStep
+
+sys.path.append(os.path.abspath("./aml_service/experiment"))
```
Comment (ricardoserradas) [inline aml_service/pipelines/train_pipeline.py]: Hi @sptramer 

This refactoring was to comply with the rule [E042](https://lintlyci.github.io/Flake8Rules/rules/E402.html), I assume.

This structure unfortunately will break the code execution, as we first need to append the path to the folder `aml_service/experiment` before importing `WorkSpace` and `AttachCompute`.

Can you please update your changes to have this piece of code as follows:

```python
sys.path.append(os.path.abspath("./aml_service/experiment"))
from attach_compute import get_compute
from workspace import get_workspace
```

**Comment by ricardoserradas** [comment]
Just added [this](https://github.com/Azure-Samples/MLOpsDatabricks/pull/3/files#r300171050) comment to the code change.

We're good to merge once this is fixed.

**Comment by sptramer** [comment]
Resolved conflicts from master and then looked like there were no PEP8 issues left - closing.

**Pull request closed without merging**